etc/lime-elements.api.md

```diff
@@ -2294,6 +2294,7 @@ export interface ListItem<T = any> {
     icon?: string | Icon;
     // @deprecated
     iconColor?: string;
+    image?: Image_2;

```

Comment:
i know this is auto generated, but it just felt weird that it says `Image_2`. Did we do something wrong, or is this correct?

Reply:
No idea it is auto generated running the command `npm run api:update`

Reply:
I don't know. I'm not sure if I've seen that in this file before. I know I've seen it somewhere, but I'm fairly sure that's been in `components.d.ts`. In this case, I find `Image_2` referenced in `dist/lime-elements.d.ts`, but I can't seem to find where it's actually coming from.

I've tried removing the existing `dist` folder and other temporary folders, and run the build and `api:update` script again, but just got the same result anyway.

It's exported as `Image` anyway, so it shouldn't be a problem:

<img width="563" alt="image" src="https://github.com/user-attachments/assets/513daf1c-0af4-457e-a018-b3fe9481c88a" />
<img width="717" alt="image" src="https://github.com/user-attachments/assets/789d9786-8581-4176-86f1-ac9aa436fd6f" />